com.arc.embeddedcdt.rxtx/pom.xml

```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
This file is referenced in root /pon.xml - you need to remove reference to it there as well. https://github.com/foss-for-synopsys-dwc-arc-processors/arc_gnu_eclipse/blob/darya-cdtnative/pom.xml#L12

Reply:
done